### DISCUSSION TITLE
Great App! Is there a "Timeline" mode?

### DISCUSSION THREAD

**Discussion by ian-droid** [vzhd1701/gridplayer#43] [Ideas]
Thanks for this nice tool! I found this while I was looking for a player for multi camera video files (like video surveillance) with overlapping and gaps, but need to be played simultaneously and synchronized. So I thought there should be a 'playlist' like file (perhaps more close to the old but dead SMIL).

However, I check the Gridplayer saved playlist file, and think maybe I need:
- New clip (from the same camera) can be loaded after the one before it finished in the same block.
- New clip (in other track) can be added into a new block and start to play at certain position of a playing clip.

Are those possible with current playlist file? I don't mind editing them manually. Or perhaps we need introduce a new play mode? (since somehow there maybe an implicit overall timeline for seeking?) 😉

Thanks again!

**Comment by vzhd1701** [comment]
> New clip (from the same camera) can be loaded after the one before it finished in the same block.

Do you mean you need an option to set an arbitrary end position for a video?

> New clip (in other track) can be added into a new block and start to play at certain position of a playing clip.

Do you mean you need support for video files with multiple video tracks?
